packages/ckeditor5-html-embed/src/htmlembedediting.js

```diff
@@ -352,6 +353,7 @@ function createDomButton( locale, type ) {
 			label: t( 'Save changes' ),
 			class: 'raw-html-embed__save-button'
 		} );
+		buttonView.bind( 'isEnabled' ).to( command, 'isEnabled' );
```

Comment:
Unfortunately, the button is destroyed (see below) here and w only use `ButtonView` to render the DOM element (by using `cloneNode()` below.

The problem with this that you can still click and execute the button. The disabled state is taken into consideration only when the button is created.

The best way to test it in the manual test is to change the editor state to ["read-only"](https://ckeditor.com/docs/ckeditor5/latest/features/read-only.html) by executing in the console:

```
editor.isReadOnly = true;
```

![](https://user-images.githubusercontent.com/247363/98508455-ca86c700-225f-11eb-99ee-2395f957691f.gif)